src/app/backend/resource/generic/controller.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package owner
+package generic
```

Comment:
I don't think generic is a good name. It doesn't say anything about what can be inside. Since it's about resource controller then `controller` would look better. It's already under resource package so we would have `resource/controller/controller.go`.